api/src/main/java/module-info.java

```diff
@@ -471,19 +475,20 @@
  * <td style="font-family:sans-serif; font-size:0.8em">Key-value<br>Wide-Column<br>Document<br>Graph</td></tr>
  *
  * <tr style="vertical-align: top; background-color:#eee"><td>{@code In}</td>
- * <td>all attribute types</td>
- * <td>Requires that the entity's attribute value be within the list that is the parameter value.</td>
- * <td>{@code findByNameIn(names)}</td>
+ * <td>sortable basic types</td>
+ * <td>Requires that the entity's attribute value be within the {@link Set} that is
```

Comment:
```suggestion
 * <td>Requires that the entity attribute value be within the {@link Set} that is
```

Reply:
Oh well, there's lots more "entity's attribute value", which I would change to just "entity attribute value".